src/operators/infinitepepo.jl

```diff
@@ -187,11 +187,11 @@ function _dag(O::PEPOTensor)
 end
 
 """
-    dagger(O::InfinitePEPO)
+    adjoint(O::InfinitePEPO)
 
-Create the dagger of a PEPOTensor such that `InfinitePEPO(dagger(O))` is the adjoint of `InfinitePEPO(O)` with respect to the physical action of the PEPO on a PEPS.
+Create the adjoint of an InfinitePEPO.
 """
-function dagger(O::InfinitePEPO)
+function TensorKit.adjoint(O::InfinitePEPO)
```

Comment:
```suggestion
function Base.adjoint(O::InfinitePEPO)
```
This function is actually defined in base, otherwise this is some kind of strange method piracy.
Given that this is the case, and that this docstring doesn't really provide more information than the Base docstrings I would also just drop the docstring

Reply:
I agree the docstring doesn't add much, but I would prefer to exapand it rather than remove it. The adjoint with respect to the physical action of a PEPO on a PEPS is not that trivial I think, especially since both physical PEPO tensor indices are in the codomain from a tensormap point of view.